lms/static/sass/views/_login-register.scss

```diff
@@ -3,22 +3,81 @@
 @import '../base/grid-settings';
 @import "neat/neat"; // lib - Neat
 
-.login-register {
+@media (min-width: 768px) {
+    .enterprise-content {
+        width: 20%;
+        float: left;
+        height: 100%;
+    }
+
+    .border-left {
+        border-left: 1px solid #d9d9d9;
+    }
+}
+
+@media (max-width: 767px) {
+    .enterprise-content {
+        margin: auto auto;
+        display: block;
+    }
+
+    img {
+        display: none;
```

Comment:
This is actually a great catch - it turns out we _don't_ want the EC logo displayed (the UI requirements changed over time, and the ticket didn't get updated), but it shouldn't be getting applied to other images. Long story short, I need to make this a more specific selector.